script.js

```diff
@@ -0,0 +1,18 @@
+window.onload = function() {
+
+const synth = new Tone.PolySynth(2, Tone.Synth, {
+    oscillator : {
+          type : "square"
+      }
+  }).toMaster();
+   
+document.querySelector('tone-keyboard').addEventListener('noteon', e => {
+    synth.triggerAttack(e.detail.name)
+  })
+  
+  document.querySelector('tone-keyboard').addEventListener('noteoff', e => {
```

Comment:
I don't believe that addEventListener has a type 'noteoff'. Is this part of the tonejs UI library that you included? When I click on a key a noise starts, but it doesn't seem to end. I am unsure if that was the intent or not.